cookbook/core/containerization/multi_images.py

```diff
@@ -99,10 +71,11 @@
 
 dataset_url = "https://raw.githubusercontent.com/harika-bonthu/SupportVectorClassifier/main/datasets_229906_491820_Fish.csv"
 
+
 # %%
 # Define a task that fetches data and splits the data into train and test sets.
 @task(
-    container_image="ghcr.io/flyteorg/flytecookbook:core-with-sklearn-baa17ccf39aa667c5950bd713a4366ce7d5fccaf7f85e6be8c07fe4b522f92c3"
```

Comment:
where is the `trainer_image` variable defined? this needs to be something "{{ .image.trainer_image.default }}:core-{{ .image.trainer_image.version }}" no?